examples/with-ant-design/app/layout.tsx

```diff
@@ -0,0 +1,17 @@
+import type { PropsWithChildren } from 'react'
+import { RootStyleRegistry } from './antd'
+
+export default function RootLayout({ children }: PropsWithChildren) {
```

Comment:
Why `RootLayout` is not used? 
Can you add an example of how it should be used?

Reply:
This example is mainly to extract css styles to avoid initial rendering without styles or oversized generation volume. If you want to know more details, you can read this: https://github.com/emotion-js/emotion/issues/2928.

Reply:
Sorry, but this is not an answer to my question. Can you explain why `RootLayout` is not used?